Common/src/geometry/CGeometry.cpp

```diff
@@ -56,6 +56,9 @@ CGeometry::CGeometry(void) {
   nPoint     = 0;
   nPointNode = 0;
   nElem      = 0;
+  Global_nPointDomain = 0;
+  Global_nPoint = 0;
+  nPointDomain = 0;
```

Comment:
Do these have s.th. to do with this PR?

Reply:
Yes, dry run was not working when MG was enabled because of that